File: seeker/snippet/palindrome.py
#date: 2022-06-20T16:59:43Z
#url: https://api.github.com/gists/796c0917e9ef415f9c0f083804441697
#owner: https://api.github.com/users/HousniBouchen

n = input("Type a string: ")
N=len(n)-1
i=0
while i<=len(n)/2:
    if n[i] != n[N]:
        break
    i=i+1
    N=N-1

if i > len(n)/2:
    print(n," is palindrome.\n")
else:
    print(n, " is not palindrome.\n")

File: seeker/snippet/Palindrome.java
//date: 2023-03-14T16:56:33Z
//url: https://api.github.com/gists/8d29614b0be985d4904f7d07014b1109
//owner: https://api.github.com/users/mahesh504

Class Palindrome{
    public static void main(String[] args)  {
        System.out.println("Enter a number");
        Scanner scanner = new Scanner(System.in);
        int digit,result=0,input, num = scanner.nextInt();
        input = num;
        while(num != 0){
            digit = num % 10;
            result = result * 10 + digit;
            num = num / 10 ;
        }

        System.out.println(result == input ? "Its palindrom": "its not a palindrom");
    }
}